thoth/storages/graph/dgraph.py

```diff
@@ -2029,6 +2053,79 @@ def sync_analysis_result(self, document: dict) -> None:
         self._python_sync_analysis_result(package_extract_run, document, environment)
         self._python_file_digests_sync_analysis_result(package_extract_run, document)
 
+    def _python_file_digests_sync_package_analysis_result(
+        self, contents: List[dict], python_artifact: PythonArtifact, filepath: str = ""
+    ) -> None:
+        """Sync python files found in the given artifact."""
+        for content in contents:
+            if content["type"] == "directory":
+                self._python_file_digests_sync_package_analysis_result(
+                    content["contents"],
+                    python_artifact,
+                    content["name"] + "/",
+                )
+            else:
+                if content["name"].endswith(".py"):
```

Comment:
@fridex As you can see I am only interested in **.py** files. However, json contains hashes for all files. Do you think we should change the logic in Packager-Analyzer, so it looks for **.py* files only or not? Thanks!

Reply:
@bissenbay keep all files in the resulting JSON and let's sync just python files into the graph database for now